src/Tizen.NUI/src/internal/Application/NUICoreBackend.cs

```diff
@@ -142,8 +142,10 @@ public void Dispose()
         /// </summary>
         public void Exit()
         {
+            Tizen.Log.Error("NUI", "NUICoreBackend.Exit() !");
```

Comment:
Is the error log level correct?

Reply:
Yes, you are correct. However, on the GBM TV target, logs at the error and fatal levels are visible. Due to the request from the GBM user, I had to include debugging logs at the error and fatal levels, although it was not ideal.

Reply:
How about adding some comments on why these logs are `Error` for the history?

Reply:
thank you for the review.
I have added some comments by following your review.

Reply:
Thanks for updating it ;)